CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ All notable changes to **ValueStringBuilder** will be documented in this file. T
 
 ## [Unreleased]
 
+## [0.7.1] - 2023-08-22
```

Comment:
Almost there - you can omit the date and version number and just add the `### Added` part under `Unreleased`.

The idea is taken from [keep-a-changelog](https://keepachangelog.com/en/1.0.0/). I have a release action that takes everything under `Unreleased` and puts it into the release notes of the github page. The version and date will be filled by that action.

Reply:
Done. Removed the date & version number.